tests/mod.rs

```diff
@@ -0,0 +1,75 @@
+use assert_cmd::{assert::Assert, Command};
+
+const MOUNTPOINT: &str = "/tmp/rmnt";
+
+fn create_mountpoint_dir() {
+    Command::new("/bin/mkdir").args(["-p", MOUNTPOINT]).output().unwrap();
+}
+
+fn remove_mountpoint_dir() {
+    Command::new("/bin/rm").args(["-Rf", MOUNTPOINT]).output().unwrap();
+}
+
+fn run_rfs_detached() -> Assert {
+    Command::new("echo").args(["user_allow_other", ">>", "/etc/fuse.conf"]).output().unwrap();
```

Comment:
Come on! You can just open the file, write, and close it instead of doing an echo. Why do you keep running commands instead of doing the things grammatically ?